Data Structures/Trie/trie.cpp

```diff
@@ -0,0 +1,129 @@
+#include<iostream>
+#define ALPHABET_SIZE 26   //defining the alphabet size of the english language
+using namespace std;
+
+//a single trienode definition
+class TrieNode{
+    //creating array for holding references to child nodes
+    public:
+    TrieNode *child[ALPHABET_SIZE];
+    //checks whether the current node is the including end of word
+    bool EndOfWord;
```

Comment:
Follow [Coding Standards](https://github.com/sahilbansal17/Competitive_Coding/wiki/Coding-Standards#naming-convention). Fix this in complete code.